interpreter/k8s/src/main/scala/io/buoyant/interpreter/k8s/IstioInterpreter.scala

```diff
@@ -28,25 +29,42 @@ object IstioInterpreter {
    /#/io.l5d.k8s.istio/default/reviews/version:v1/http
   */
 
-  def apply(routeManager: Unit, istioNamer: Namer): NameInterpreter = {
+  private[this] def mkDentry(name: String, route: RouteRule): Option[Dentry] =
+    route.destination.flatMap { dest =>
+      dest.split('.') match {
+        case Array(svc, ns, _*) =>
+          val branches = route.route.map { weightedDest =>
+            val labels = weightedDest.tags.toSeq
+              .sortBy(_._1)
+              .map { case (k, v) => s"$k:$v" }
+              .mkString("::")
+            val labelSegment = if (labels.isEmpty) "::" else labels
+            NameTree.Weighted(
+              weightedDest.weight.getOrElse(0).toDouble,
+              NameTree.Leaf(Path.read(s"/#/io.l5d.k8s.istio/$ns/$svc/$labelSegment"))
+            )
+          }
+          val prefix = Dentry.Prefix.read(s"/svc/route/$name")
+          Some(Dentry(prefix, NameTree.Union(branches: _*)))
+        case _ =>
+          None
+      }
+    }
+
+  def apply(routeManager: RouteManager, istioNamer: Namer): NameInterpreter = {
```

Comment:
IstioNamers still can be configured directly if you wanted to use one standalone, but for the IstioInterpreter it has an IstioNamer that it uses internally that is not configured directly.  This is similar to how in a linkerd config that uses namerd, namers won't be configured directly in the linkerd config because the namers are "internal" to namerd.

We pass the istioNamer in here so we can pass it to the ConfiguredDtabNamer.